.github/dependabot.yml

```diff
@@ -10,6 +10,16 @@ updates:
     schedule:
       interval: "daily"
     open-pull-requests-limit: 5
+  - package-ecosystem: "gomod"
+    directory: "modules/common/tools/"
```

Comment:
I'd change it to whole `modules/common` dir. We include K8s dependencies in `modules/api` anyway. In my opinion it makes sense as we will be notified about any releases.

Reply:
Unfortunately dependabot does not check recursively. So if we want to check whole `module/common` directory, we need to add one more directive for `module/common/certificates`.

Reply:
Let's add it then.

Reply:
Got it. Done.